CMakeLists.txt

```diff
@@ -264,7 +264,7 @@ if (${TRITON_PYTORCH_DOCKER_BUILD})
     COMMAND docker cp -L pytorch_backend_ptlib:/usr/local/${LIB_DIR}/libopencv_features2d.so libopencv_features2d.so
     COMMAND docker cp -L pytorch_backend_ptlib:/usr/local/${LIB_DIR}/libopencv_flann.so libopencv_flann.so
     COMMAND /bin/sh -c "docker cp pytorch_backend_ptlib:$<IF:$<BOOL:${RHEL_BUILD}>,/usr/lib64/libpng16.so.16.34.0,/usr/lib/${LIBS_ARCH}-linux-gnu/libpng16.so.16.37.0> libpng16.so"
-    COMMAND /bin/sh -c "docker cp pytorch_backend_ptlib:$<IF:$<BOOL:${RHEL_BUILD}>,/usr/lib64/libjpeg.so.62.2.0,/usr/lib/${LIBS_ARCH}-linux-gnu/libjpeg.so.8.2.2> libjpeg.so"
+    COMMAND /bin/sh -c "docker cp -L pytorch_backend_ptlib:$<IF:$<BOOL:${RHEL_BUILD}>,/usr/lib64/libjpeg.so.62,/usr/local/lib/python3.10/dist-packages/torchvision/libjpeg.so.62> libjpeg.so"
```

Comment:
Forgot to ask: has this directory been populated by the time the execution arrives here? i.e. has /usr/local/lib/python3.10/dist-packages/torchvision been populated by installing the torchvision wheel?

Reply:
Sorry, must've missed this. It would have been, yes. But we're now using Misha's changes linked below.